packages/vscode-extension/scripts/download-sim-server-release-assets.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# This script downloads pre-built signed release versions of the simulator-server
+# binaries and places them in the build location for packaging.
+
+set -e
+
+# execute this from the vscode-extension package directory
+cd "$(dirname "$0")/.."
+
+# take output directory from first argument or default to out - relative to the vscode-extension package location
+output_dir="${1:-out}"
+
+# second argument is the tag of the simulator-server release to download, defaults to latest
+sim_server_tag="${2:-latest}"
+
+# Create the target directory if it doesn't exist
+mkdir -p "$output_dir"
+
+echo "Downloading simulator-server assets for tag $sim_server_tag"
+
+# List release assets using GitHub API
+release_info=$(curl -s "https://api.github.com/repos/software-mansion-labs/simulator-server-releases/releases/$sim_server_tag")
```

Comment:
are these going to be rate limited? was wondering if we should perhaps stick to using github via CLI where we can authenticate requests with GITHUB_TOKEN or use `GITHUB_TOKEN` here to make the curl request

Reply:
I didn't encounter any rate limits, but since it happened to @filip131311 then yeah, It makes sense to restore GH CLI and create a PAT for the CI for authorization. Having this set up, I'm not sure if we even need a public repo for `simulator-server` releases.